go/common/identity/identity.go

```diff
@@ -82,88 +66,10 @@ type Identity struct {
 	// connecting to the sentry node's control connection.  It is never rotated.
 	TLSSentryClientCertificate *tls.Certificate
 
-	// DoNotRotateTLS flag is true if we mustn't rotate the TLS certificates below.
-	DoNotRotateTLS bool
-
 	// tlsSigner is a node TLS certificate signer.
 	tlsSigner signature.Signer
```

Comment:
I think this (and certificate?) no longer needs an accessor function given it is no longer modified and so can be exposed directly similar to the other signers? This also probably means that the lock itself and mutation methods are no longer needed?